notebooks/analyze_model_output.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
<p>A format that I have started to follow in other notebooks is:</p><h1>Title</h1><p><strong>Author</strong>: &lt;Name&gt; (&lt;email&gt;)</p><p><strong>Description:</strong>&nbsp;&lt;what you wrote in the PR&gt;</p><p> </p><p>Do you think it would be useful to add to notebooks like this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>good idea</p>